### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by pokey**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -144,7 +144,12 @@ jobs:
             version=$(cat .bumpversion.cfg | awk '/current_version / {print $3}')
             python setup.py register -r pypi
             python setup.py sdist
-            twine upload dist/microcosm-sagemaker-${version}.tar.gz
+            twine upload --repository pypi dist/microcosm-sagemaker-${version}.tar.gz
+            version=$(python setup.py --version)
+            globality-build dependency-update \
+                --package-name microcosm_sagemaker \
```
Comment (KensoDev) [inline .circleci/config.yml]: This should be `microcosm-sagemaker` actually.

File: .globality/build.json [hunk 2]
```diff
@@ -1,11 +1,14 @@
 {
   "params": {
+    "commitbot": {
```
Comment (KensoDev) [inline .globality/build.json]: If you enable commitbot, you should also add the permission for it in `terraform-github`.

File: .circleci/config.yml (hunk 1 above)
Comment (pokey) [reply .circleci/config.yml]: Should this use `{{ name }}` then?  Or should it be something like

```py
{{ inflection.dasherize(import_name) }}
```

File: .globality/build.json (hunk 2 above)
Comment (pokey) [reply .globality/build.json]: globality-corp/terraform-github#31

**Pull request merged**